File: TDD Boot Camp/Part 01/Code/OOPrinciples/src/com/oop/telldonotask/good/Tester.java
package com.oop.telldonotask.good;

public class Tester {
	public Unit convertToCentiMeters() {
		Unit actual = new Unit(10,UnitType.MILIMETERS);
		return actual.convertTo(UnitType.CENTIMETERS);
	}
	public Unit convertToMeters() {
		Unit actual = new Unit(10,UnitType.CENTIMETERS);
		return actual.convertTo(UnitType.METERS);
	}
}

File: TDD Boot Camp/Part 01/Code/C#/OOPrinciples/TellDonotAsk/Good/Tester.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TellDoNotAsk.Good
{
    public class Tester
    {
        public Unit ConvertToCentiMeters()
        {
            Unit actual = new Unit(10, UnitType.MILIMETERS);
            return actual.ConvertTo(UnitType.CENTIMETERS);
        }
        public Unit ConvertToMeters()
        {
            Unit actual = new Unit(10, UnitType.CENTIMETERS);
            return actual.ConvertTo(UnitType.METERS);
        }
    }
}
